infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -135,11 +135,11 @@
 
   post_tasks:
     - name: waiting for the monitor to join the quorum...
-      command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} quorum_status --format json"
+      command: "{{ container_binary }} run --rm  -v /etc/ceph:/etc/ceph -v /run/ceph:/run/ceph --entrypoint=ceph {{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }} daemon mon.{{ mon_host }} quorum_status --format json"
```

Comment:
you need to add the `:z` flag to the bind mounts and readd the `cluster` variable